new_client/src/components/projects/Projects.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import { Container, Grid, Typography } from "@material-ui/core";
+import ProjectGroup from './ProjectGroupCard';
+
+const Projects = () => {
+    const [domains, setDomains] = useState([]);
+
+    useEffect(() => {
+        axios
+            .get(process.env.REACT_APP_API + "/projects")
+            .then((res) => setDomains(res.data));
+    }, [])
+
+    return (
+        <Container maxWidth="lg">
+        <Grid container spacing={1}>
+        <Grid item xs={12}>
+        <Typography align="center" variant="h4" gutterBottom="true"> Project Categories </Typography>
```

Comment:
`gutterBottom` is a boolean prop, so only true (without the "") or just `gutterbottom` should suffice.